README.md

```diff
@@ -119,7 +119,7 @@ Process:
      - Check manifest validity
      - Provide links to relevant diffs
      - Check to see if the GitHub user submitting the PR is a maintainer of the Addon version being submitted.
-   - On Appveyor CI, install most recently supported version of NVDA supported by addon, install the addon, restart NVDA and check for errors.
+   - On Appveyor CI, install most recently supported version of NVDA supported by addon, install the addon, restart NVDA and check for errors. If the add-on contains an `installTasks.py` file with an `onInstall()` function which may require input at installation, the `installTasks` file may be skypped to avoid false errors.
```

Comment:
I'd prefer not to get into the details of things like this, it's really not core to the proposal, and really just a suggestion of the "type" of things that could be done. Perhaps instead it should just be made more clear that this is a "suggestion".

```suggestion
   - Suggestion: On Appveyor CI, install most recently supported version of NVDA supported by addon, install the addon, restart NVDA and check for errors. 
```